src/index.html

```diff
@@ -8,6 +8,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star star"></div>
+      <div class="stars__star star"></div>
+      <div class="stars__star star"></div>
+      <div class="stars__star star"></div>
+      <div class="stars__star star"></div>
+    </div>
+
+    <div class="stars stars--1">
+      <div class="stars__star star star-active"></div>
+      <div class="stars__star star"></div>
+      <div class="stars__star star"></div>
+      <div class="stars__star star"></div>
+      <div class="stars__star star"></div>
```

Comment:
don't highligh start in html code, do it using css. all stars must have the same classes